modules/jaeger-sidecar-manual.adoc

```diff
@@ -30,7 +30,7 @@ spec:
             - containerPort: 8080
               protocol: TCP
         - name: jaeger-agent
-          image: registry.redhat.io/distributed-tracing/jaeger-agent-rhel7:<version>
+          image: acme/myapp:myversion
```

Comment:
This change needs to be reverted - the `image: acme/myapp:myversion` needs to go under `name: example-app`.

Reply:
also the `image: ` field has not been added back into the automatic sidecar injection yaml.